### PR TITLE
Add an option to include prerelease updates.

### PR DESCRIPTION
**Pull request by pixlwave**
Hope this is a useful addition.

### CODE REVIEW

File: AppUpdater.swift [hunk 1]
```diff
@@ -183,9 +185,9 @@ extension Release: Comparable {
 }
 
 private extension Array where Element == Release {
-    func findViableUpdate(appVersion: Version, repo: String) throws -> Release.Asset? {
-        let properReleases = filter{ !$0.prerelease }
-        guard let latestRelease = properReleases.sorted().last else { return nil }
+    func findViableUpdate(appVersion: Version, repo: String, prerelease: Bool) throws -> Release.Asset? {
+        let suitableReleases = prerelease ? self : filter{ !$0.prerelease }
```
Comment (pixlwave) [inline AppUpdater.swift:189]: Is this too concise at the expense of readability?

File: AppUpdater.swift [hunk 2]
```diff
@@ -17,6 +17,8 @@ public class AppUpdater {
     var slug: String {
         return "\(owner)/\(repo)"
     }
+    
+    public var prerelease = false
```
Comment (pixlwave) [inline AppUpdater.swift]: Not sure about the best language to use, e.g. should it be `updater.prerelease` or more like `updater.includePrereleases` or `updater.allowPrerelease`?

File: AppUpdater.swift (hunk 1 above)
Comment (mxcl) [reply AppUpdater.swift:189]: I'm cool with it.

File: AppUpdater.swift (hunk 2 above)
Comment (mxcl) [reply AppUpdater.swift]: I think `allowPrereleases` is most Swifty.

Comment (pixlwave) [reply AppUpdater.swift]: Updated.

**Comment by pixlwave** [comment]
Thanks 🙌🏻

**Pull request merged**